tla/governance-ledger/Governance.tla

```diff
@@ -664,13 +665,13 @@ process ( Merge_Neurons \in Merge_Neurons_Process_Ids )
                 send_request(self, OP_QUERY_BALANCE, balance_query(neuron[target_neuron_id].account));
             };
         };
-    MergeNeurons3:
+    MergeNeurons4:
```

Comment:
I can't find the balance query stuff in the code any more, but I find that the code has also unfortunately become a lot more difficult to understand; could you point me to where this query happens?

Reply:
It does not query ledger. I think this is an intentional divergence from the implementation in the TLA model. The additional query seems to be added to catch a violation of an invariant.